build/creator/ZcToken.sol

```diff
@@ -17,7 +17,7 @@ contract ZcToken is Erc20, IERC5095 {
     /// @dev address of a cToken
     address public immutable cToken;
     /// @dev address and interface for an external custody contract (necessary for some project's backwards compatability)
-    IRedeemer public immutable redeemer;
+    address public immutable redeemer;
```

Comment:
this contract was already expecting this to be an address and uses the `Interface(address).method()` pattern.

this is likely addressing something in the audit somewhere.

Reply:
I believe this also generally inline with our pattern of storing addresses and not interfaces anyways, right?

Reply:
yeah